Improved eventing getting-started guide (#14739)
kyma-project/kyma: docs/02-get-started/04-trigger-workload-with-event.md

## Create a Subscription

> Next, to subscribe to an event, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe for an event of type `order.received.v1`. 

Reviewer: ```suggestion
Next, to subscribe to an event, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe to an event of the type `order.received.v1`. 
```

Author: I updated the line as:
```
Next, to subscribe to events, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe to events of the type `order.received.v1`.
```

> All the events published on this event type will be forwarded to the `Sink` (i.e. HTTP endpoint) defined in the Subscription's Spec.

Reviewer: ```suggestion
All the published events of this type are then forwarded to an HTTP endpoint called `Sink`. You can define this endpoint in the Subscription's spec.
```

## Trigger the workload with an event

> We created the `lastorder` Function, and subscribed for `order.received.v1` events by creating a Subscription CR. Now it's time to publish an event and trigger the Function. In this example, we'll port-forward the Kyma Eventing Service to localhost. 

Reviewer: ```suggestion
We created the `lastorder` Function and subscribed to the `order.received.v1` event by creating a Subscription CR. Now it's time to publish your event and trigger the Function. In this example, we'll port-forward the Kyma Eventing Service to localhost. 
```

Reviewer: We're talking about one event here, is that correct?

Author: yes
